CRM/Price/BAO/PriceField.php

```diff
@@ -94,22 +94,15 @@ public static function create(&$params) {
     }
     $optionsIds = array();
     $maxIndex = CRM_Price_Form_Field::NUM_OPTION;
-
-    if ($priceField->html_type == 'Text') {
+    $priceField2 = civicrm_api3('price_field', 'getsingle', array('id' => $priceField->id));
+    if ($priceField2['html_type'] == 'Text') {
       $maxIndex = 1;
-
-      $fieldValue = new CRM_Price_DAO_PriceFieldValue();
-      $fieldValue->price_field_id = $priceField->id;
-
-      // update previous field values( if any )
-      if ($fieldValue->find(TRUE)) {
-        $optionsIds['id'] = $fieldValue->id;
-
-        //Update price_field_value label when edited inline.
-        if (!empty($params['id']) && $priceField->label != $fieldValue->label) {
-          $fieldValue->label = $priceField->label;
-          $fieldValue->save();
-        }
```

Comment:
Removal of this part seems to be causing a minor regression of https://issues.civicrm.org/jira/browse/CRM-19741. Should have written a test :(. Is this necessary to be removed?